faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayBinaryFormat.java

```diff
@@ -0,0 +1,9 @@
+package com.faforever.commons.replay;
+
+public sealed interface ReplayBinaryFormat {
+
+  record WithContext (byte[] bytes) implements ReplayBinaryFormat {}
+
+  record BinarySCFA(byte[] bytes) implements ReplayBinaryFormat {}
+
+}
```

Comment:
I don't think these classes are necessary anymore. As it is know they are only used as a wrapper around the bytes and only a member of the ReplayContainer. But they are only created and used inside the ReplayLoader without being exposed publicly. So I think the ReplayLoader can just handle the bytes appropriately without these types.

Reply:
Processed in #143